### ISSUE TITLE
zcashd no connection?

### ISSUE THREAD

**Issue by nekkidtruth** [ZcashFoundation/zecwallet#231]
Just installed using .msi file. Added exclusions in AV and Malware related software.

I do not check off Fast Sync. It goes through the process of downloading some files and then it looks like it's going to start but instead I get:

Could not connect to zcashd

This is with the "FullNode" release.

**Comment by adityapk00** [comment]
Hello.

There was an issue with the previous zcashd, which has since been fixed. Can you try:
1. Uninstall the existing Zecwallet Fullnode
2. Download the latest msi: (https://github.com/ZcashFoundation/zecwallet/releases/download/v0.9.1/Zecwallet.FullNode.0.9.1.msi)
3. Install and run

Note that this is the fullnode release, which will download the entire blockchain (~25 GB), which can take several days. 

If you want to get started immediately, I'd recommend downloading the lightwallet: https://www.zecwallet.co

**Comment by nekkidtruth** [comment]
Thanks for the response. That was indeed the file I downloaded and installed. I actually resolved the issue by physically downloading zcashd.exe from master and I had to create \bin\win folders in \Roaming\zecwallet\ as the file did not exist.

I restarted ZecWallet and it still had the same error. I manually went into \bin\win through command line and started zcashd.exe directly and the ZecWallet immediately started working. However, I don't believe this is working as intended?
